src/main/java/com/redislabs/redisai/Keyword.java

```diff
@@ -16,12 +15,19 @@
   BATCHSIZE,
   MINBATCHSIZE,
   BACKENDSPATH,
-  LOADBACKEND;
+  LOADBACKEND,
+  LOAD,
+  PERSIST,
+  PIPE;
 
   private final byte[] raw;
 
   Keyword() {
-    raw = SafeEncoder.encode(this.name());
+    if (this.name() == "PIPE") {
+      raw = SafeEncoder.encode("|>");
+    } else {
+      raw = SafeEncoder.encode(this.name());
+    }
   }
```

Comment:
```java
PIPE("|>");

private final byte[] raw; 

Keyword() {
  this(this.name());
}

Keyword(String word) {
       raw = SafeEncoder.encode("|>"); 
} 
```

Reply:
@gkorland that implies:
```

public enum Keyword implements ProtocolCommand {
  INPUTS("INPUTS"),
  OUTPUTS("OUTPUTS"),
  META("META"),
  VALUES("VALUES"),
  BLOB("BLOB"),
  SOURCE("SOURCE"),
  RESETSTAT("RESETSTAT"),
  TAG("TAG"),
  BATCHSIZE("BATCHSIZE"),
  MINBATCHSIZE("MINBATCHSIZE"),
  BACKENDSPATH("BACKENDSPATH"),
  LOADBACKEND("LOADBACKEND"),
  LOAD("LOAD"),
  PERSIST("PERSIST"),
  PIPE("|>");

  private final byte[] raw;

  Keyword(String s) {
    raw = SafeEncoder.encode(s);
  }
(...)
```

Reply:
I think his suggestion is that you should add two constructors. :)
The complete code should be:
```
public enum Keyword implements ProtocolCommand {
  INPUTS,
  OUTPUTS,
  META,
  VALUES,
  BLOB,
  SOURCE,
  RESETSTAT,
  TAG,
  BATCHSIZE,
  MINBATCHSIZE,
  BACKENDSPATH,
  LOADBACKEND,
  LOAD,
  PERSIST,
  PIPE("|>");

  private final byte[] raw;

 Keyword() {
   this(this.name());
 }

  Keyword(String s) {
    raw = SafeEncoder.encode(s);
  }
(...)
```

Reply:
Your totally right @gkorland and @dengliming . was completely missing it =) now fixed